<ui version="4.0" >
 <class>Panel</class>
 <widget class="QWidget" name="Panel" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>206</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Panel</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="3" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Size and Color</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QToolButton" name="sizeUp" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QToolButton" name="sizeDown" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QToolButton" name="color" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Alignment</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QToolButton" name="left" >
     <property name="text" >
      <string>...</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QToolButton" name="center" >
     <property name="text" >
      <string>...</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2" >
    <widget class="QToolButton" name="right" >
     <property name="text" >
      <string>...</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Type</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QToolButton" name="bold" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="QToolButton" name="italic" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" >
    <widget class="QToolButton" name="underline" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Style</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0" >
    <widget class="QToolButton" name="style1" >
     <property name="text" >
      <string>...</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="1" >
    <widget class="QToolButton" name="style2" >
     <property name="text" >
      <string>...</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="2" >
    <widget class="QToolButton" name="style3" >
     <property name="text" >
      <string>...</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3" >
    <spacer name="verticalSpacer" >
     <property name="spacerName" stdset="0" >
      <string>verticalSpacer</string>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>131</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
